app/domain/logs.rb

```diff
@@ -591,106 +591,106 @@ module AuthnJwt
         code: "CONJ00122D"
       )
 
       NotConfiguredMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Mandatory claims is not configured",
+        msg:  "Mandatory claims is not configured",
         code: "CONJ00123D"
       )
 
       FetchedMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully fetched mandatory claims '{0-mandatory-claims}'",
+        msg:  "Successfully fetched mandatory claims '{0-mandatory-claims}'",
         code: "CONJ00124I"
       )
 
       ParsingMappingClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Parsing mapping claims value '{0-mapping-claims}'",
+        msg:  "Parsing mapping claims value '{0-mapping-claims}'",
         code: "CONJ00125D"
       )
 
       ParsedMappingClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully parsed mapping claims '{0-mapping-claims-table}'",
+        msg:  "Successfully parsed mapping claims '{0-mapping-claims-table}'",
         code: "CONJ00126D"
       )
 
       ClaimMapDefinition = ::Util::TrackableLogMessageClass.new(
-        msg: "Mapping annotation name '{0-annotation-value}' to the claim name '{1-claim-name}'",
+        msg:  "Mapping annotation name '{0-annotation-value}' to the claim name '{1-claim-name}'",
         code: "CONJ00127D"
       )
 
       FetchingMappingClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Fetching mapping claims...",
+        msg:  "Fetching mapping claims...",
         code: "CONJ00128D"
       )
 
       NotConfiguredMappingClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Mapping claims are not configured",
+        msg:  "Mapping claims are not configured",
         code: "CONJ00129D"
       )
 
       FetchedMappingClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully fetched mapping claims '{0-mapping-claims}'",
+        msg:  "Successfully fetched mapping claims '{0-mapping-claims}'",
         code: "CONJ00130I"
       )
 
       CreateContraintsFromPolicy = ::Util::TrackableLogMessageClass.new(
-        msg: "Creating constraints from policy...",
+        msg:  "Creating constraints from policy...",
         code: "CONJ00131I"
       )
 
       CreatedConstraintsFromPolicy = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully created constraints from policy",
+        msg:  "Successfully created constraints from policy",
         code: "CONJ00132I"
       )
 
       ConvertingClaimAccordingToMapping = ::Util::TrackableLogMessageClass.new(
-        msg: "Converting claim '{0-claim-name}' to '{1-annotation-name}'...",
+        msg:  "Converting claim '{0-claim-name}' to '{1-annotation-name}'...",
         code: "CONJ00133D"
       )
 
       MandatoryClaimsToBeChecked = ::Util::TrackableLogMessageClass.new(
-        msg: "Mandatory claims to be checking in host are '{0-mandatory-claims}'",
+        msg:  "Mandatory claims to be checking in host are '{0-mandatory-claims}'",
```

Comment:
don;t understand

Reply:
@nessiLahav @tzheleznyak Could you please to explain the meaning of the message